hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/SubsystemExtension.java

```diff
@@ -58,7 +58,7 @@ public void initialize(ExtensionContext context) {
                 MAJOR_VERSION, MINOR_VERSION, MICRO_VERSION);
 
         // This subsystem should be runnable on a host
-        subsystem.setHostCapable();
+        //subsystem.setHostCapable();
```

Comment:
Note - this is bad. If we can't have this API in EAP 6.x, it means we can't run the agent in the host controllers AFAIK.